remote-api/src/test/java/org/alfresco/rest/api/impl/CategoriesImplTest.java

```diff
@@ -252,6 +254,33 @@ public void testGetCategoryById_includeCount()
             .isEqualTo(1);
     }
 
+    @Test
+    public void testGetCategoryById_includePath()
+    {
+        final QName categoryQName = createCmQNameOf(CATEGORY_NAME);
+        final NodeRef parentCategoryNodeRef = createNodeRefWithId(PARENT_ID);
+        final ChildAssociationRef parentAssociation = createAssociationOf(parentCategoryNodeRef, CATEGORY_NODE_REF, categoryQName);
+        final Path mockPath = new Path();
```

Comment:
Sorry to be picky - I like the way we can connect `mockPath` factors with final assertions - but maybe `mockRootLevel` and `mockChildLevel` could be constants at the class level and whole `mockPath` creation could be extracted to a method?

Reply:
Sure, added`@Override` and cleaned up mocking path